### PR TITLE
Project Nav as list

### PR DESCRIPTION
**Pull request by lancesnider**
Package: app-project

This is related to: https://github.com/zooniverse/front-end-monorepo/issues/667 

Describe your changes:

For accessibility reasons, this updates the project nav to use `ul` and `li`. 

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?

### CODE REVIEW

File: packages/app-project/src/components/ProjectHeader/components/Nav/components/NavLink/NavLink.js [hunk 1]
```diff
@@ -16,20 +16,24 @@ const StyledAnchor = styled(Anchor)`
...
+function NavLink(props) {
   const { href, router, text } = props
   const isActive = router.asPath.includes(href)
   return (
-    <StyledAnchor href={href} isActive={isActive}>
-      <StyledSpacedText color='white' weight='bold'>
-        {text}
-      </StyledSpacedText>
-    </StyledAnchor>
+    <Box as="li">
```
Comment (rogerhutchings) [inline packages/app-project/src/components/ProjectHeader/components/Nav/components/NavLink/NavLink.js]: I'd rather this was moved to the consuming component, as I don't want `NavLink` to make assumptions on how it's rendered

File: packages/app-project/src/components/ProjectHeader/components/DropdownNav/DropdownNav.js [hunk 2]
```diff
@@ -43,34 +45,38 @@ class DropdownNav extends React.Component {
...
-                </SpacedText>
-              )}
-              theme={this.props.theme}
-            />
-          </Link>
-        ))}
+        <Box as="ul">
+          {this.props.navLinks.map(navLink => (
+            <Box as="li">
+              <Link href={navLink.href} key={navLink.href} passHref>
```
Comment (rogerhutchings) [inline packages/app-project/src/components/ProjectHeader/components/DropdownNav/DropdownNav.js]: The `key` prop needs to be moved to `<Box as='li'>`

File: packages/app-project/src/components/ProjectHeader/components/Nav/components/NavLink/NavLink.js (hunk 1 above)
Comment (rogerhutchings) [reply packages/app-project/src/components/ProjectHeader/components/Nav/components/NavLink/NavLink.js]: This will also fix the broken test

**Comment by lancesnider** [comment]
Thanks @rogerhutchings! I'll fix this up this weekend.

**Comment by lancesnider** [comment]
@srallen Here are the fixes: https://github.com/zooniverse/front-end-monorepo/pull/718/commits/35f6113a5f532d3656bdc4aac17f54ed543c3e15 

A ran the linter in a separate commit.

**Comment by rogerhutchings** [comment]
Thanks!

**Pull request merged**